polyfills/DOMTokenList/prototype/replace/config.toml

```
@@ -0,0 +1,14 @@
+dependencies = [ "DOMTokenList" ]
+license = "MIT"
+spec = "https://dom.spec.whatwg.org/#domtokenlist"
+
+[browsers]
+android = "<81"
+edge = "<18"
+firefox = "<61"
+ie = "*"
+ios_saf = "<12"
+op_mob = "*"
+opera = "<54"
+safari = "<12"
+samsung_mob = "<9"
```

I've added the browsers which were missing and updated ther versions based on the mdn data https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/replace#Browser_compatibility
```suggestion
android = "<67"
bb = "*"
chrome = "<67"
edge = "<18"
edge_mob = "<18"
firefox = "<61"
firefox_mob = "<61"
ie = "*"
ie_mob = "*"
ios_saf = "<12"
op_mini = "*"
op_mob = "<48"
opera = "<54"
safari = "<12"
samsung_mob = "<9"
```